src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -1073,6 +1171,39 @@ public static int getBitDepth(IDataSetPrx dataset) throws Error {
         return datatype.get(type);
     }
 
+    /**
+     * allows to change bit depth of the dataset
+     * (Adapted from existing function in EasyXT-Matlab)
+     *
+     * TODO discuss renaming function to setDatasetBitDepth()
+     * TODO discuss parameter as int, thought it would be nice to directly use it with {@link #getBitDepth(IDataSetPrx)} and imp.getBitDepth()
+     *
+     * @param bit_depth
+     * @throws Error
+     */
+    public static void setDataType(int bit_depth) throws Error {
```

Comment:
agree that we could change it to setDatasetbitDepth

Also split into 
```
public static void setDatasetBitDepth(IDataSetPrx  vDataSet, int bit_depth) throws Error {
```

and for your needs
```
public static void setCurrentDatasetBitDepth( int bit_depth ) throws Error {
    setDatasetBitDepth( getCurrentDataSet(), bit_depth )
```

Reply:
55fec5c1c2a22e73fc1a54157b3168358cb2dfc0